packages/account/src/Styles/account.scss

```diff
@@ -1597,3 +1597,207 @@ $MIN_HEIGHT_FLOATING: calc(
         width: auto !important;
     }
 }
+
+/** @define deactivate-account; weak */
+.deactivate-account {
+    max-width: 67.2rem;
+
+    &__information {
+        @include typeface(--paragraph-left-normal-prominent, none);
+        line-height: 1.43;
+        margin-bottom: 8px;
+
+        &--bold {
+            font-weight: bold;
+            margin-bottom: 8px;
+            margin-top: 5px;
+        }
+    }
+    &__steps {
+        margin-top: 16px;
+        @include typeface(--paragraph-left-normal-prominent, none);
+        line-height: 1.43;
+    }
+    &__title {
+        font-weight: bold;
+        margin-bottom: 8px;
+    }
+    &__content {
+        color: var(--text-general);
+    }
+    &__link {
+        color: var(--text-loss-danger);
+        font-weight: bold;
+        text-decoration: none;
+    }
+    &__button {
+        margin-top: 24px;
+        float: right;
+    }
+    .deactivate-account-reasons {
+        color: var(--text-prominent);
+        font-size: var(--text-size-xs);
+        line-height: 1.43;
+        font-weight: bold;
+
+        &__title {
+            margin-bottom: 18px;
+        }
+        &__checkbox {
+            margin-top: 10px;
+            width: fit-content;
+
+            .dc-checkbox__label {
+                font-weight: normal;
+            }
+        }
+        &__input {
+            margin: 16px 0 0;
+            max-width: 400px;
+            text-overflow: ellipsis;
+
+            > input {
+                min-height: 96px;
+            }
+        }
+        &__error {
+            color: var(--text-loss-danger);
+        }
+    }
+    @include mobile {
+        width: 92%;
```

Comment:
Can we use `width: calc(100% - offset in rem)` instead of `percentage` values?

Reply:
sure, done :+1: